### DISCUSSION TITLE
Getting 'err_unmatch_issuer' even when validation is skipped

### DISCUSSION THREAD

**Discussion by BradA1878** [tngan/samlify#498] [Q&A] [answered]
Hi,

I am setting up a SAML SSO on a Wix/Velo site. To do so, I've been following this article: https://support.wix.com/en/article/using-saml-sso-with-velo

As shown in that article, I have everything in place but cannot get the validator to "skip" validation. Here is the code snippet:

        samlify.setSchemaValidator({
            validate: (response) => {
                // implment your own or always returns a resolved promise to skip
                console.log('response', response)
                return Promise.resolve('skipped');
            }
        })

This results in a `err_unmatch_issuer` error. Can someone help me understand what I am doing wrong here?

Thanks!
~Brad

**Comment by BradA1878** [comment, marked as answer]
Disregard this, it was a case miss match with the entityID. Sorry!

**Reply by hmagdy** [reply]
How did you fix this @BradA1878 ? I am having the same problem

**Reply by BradA1878** [reply]
@hmagdy - In my case, I had a case mismatch in the entityID of the metadata. It's case-sensitive.
